src/routes/compare/+page.svelte

```diff
@@ -0,0 +1,146 @@
+            upperStr !== "PLUTO" &&
+            upperStr !== "SAGITTARIUS" &&
+            upperStr !== "SATURN" &&
+            upperStr !== "SCORPIO" &&
+            upperStr !== "SUN" &&
+            upperStr !== "TAURUS" &&
+            upperStr !== "URANUS" &&
+            upperStr !== "VENUS" &&
+            upperStr !== "VIRGO"
+            ) {
+            throw new Error("Look up failed.");
+        }
+        const element: keyof typeof Element = upperStr;
+        return Element[element];
+    }
+
+    function stringToHouseNumber(str): HouseNumber {
+        const upperStr = str.toUpperCase()
+        if (upperStr !== "HOUSE_1" && 
+            upperStr !== "HOUSE_2" &&
+            upperStr !== "HOUSE_3" &&
+            upperStr !== "HOUSE_4" &&
+            upperStr !== "HOUSE_5" &&
+            upperStr !== "HOUSE_6" &&
+            upperStr !== "HOUSE_7" &&
+            upperStr !== "HOUSE_8" &&
+            upperStr !== "HOUSE_9" &&
+            upperStr !== "HOUSE_10" &&
+            upperStr !== "HOUSE_11" &&
+            upperStr !== "HOUSE_12"
+            ) {
+            throw new Error("Look up failed.");
+        }
+        const houseNumber: keyof typeof HouseNumber = upperStr;
+        return HouseNumber[houseNumber];
+    }
+
+    function planetToPlacement(planet: PlacementData): Placement {
+        return new Placement(stringToElement(planet.sign), planet.degrees)
+    }
+
+    function houseToPlacement(strHouseNumber: string, house: PlacementData): House {
+        return new House(stringToHouseNumber(strHouseNumber), stringToElement(house.sign))
+    }
+
+    function profileToPersonType(profile: Profile): Person {
+        var currPerson = new Person(
+            profile.name,
+            planetToPlacement(profile.planets.Rising),
+            planetToPlacement(profile.planets.Moon),
+            planetToPlacement(profile.planets.Sun),
+            planetToPlacement(profile.planets.Mars),
+            planetToPlacement(profile.planets.Mercury),
+            houseToPlacement("house_2", profile.houses[1]),
+            houseToPlacement("house_3", profile.houses[2]),
+            houseToPlacement("house_6", profile.houses[5]),
+            houseToPlacement("house_10", profile.houses[9]),
+            houseToPlacement("house_11", profile.houses[10]),
+            // profile.placement.Jupiter,
+            // profile.placement.MC,
+            // profile.placement.Neptune,
+            // profile.placement.Pluto,
+            // profile.placement.Saturn,
+            // profile.placement.Uranus,
+            // profile.placement.Venus,
+        );
+
+        return currPerson;
+    }
+
+    onMount(() => {
+        const ac = new AspectCalculator();
+        const hc = new HouseCalculator();
+
+        const persons = $profiles.map(
+            (profile) => {
+                return profileToPersonType(profile)
+            }
+        )
+
+        ac.processPeople(persons[0], persons[1])
```

Comment:
@maknop as suggested by @josejulio - might be worth moving this logic to the existing profile view as a `Compare with: ` functionality.